### PR TITLE
Add licence status banner to pdf when not active

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: pages/project-version/pdf/index.jsx [hunk 1]
```diff
@@ -32,7 +34,7 @@ module.exports = settings => {
           headerTemplate: header,
           footerTemplate: footer,
           margin: {
-            top: 100,
+            top: hasStatusBanner() ? 180 : 100,
```
Comment (joefitter) [inline pages/project-version/pdf/index.jsx]: I can get on board with saving the result of the condition to a single-use variable, but having a function which always returns the same value seems particularly redundant

File: pages/project-version/pdf/views/header.jsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+import StatusBanner from './status-banner';
 
 const format = date => moment(date).format('DD MMM YY');
 
-const Header = ({ project }) => (
-  <Wrapper name="header">
+const Header = ({ project, nonce }) => (
+  <Wrapper name="header" nonce={nonce}>
     <header>
+      <StatusBanner licence={project} licenceType="ppl" />
```
Comment (joefitter) [inline pages/project-version/pdf/views/header.jsx]: Do we _always_ want a status banner? If so is the previous conditional with different padding now redundant?

Comment (joefitter) [reply pages/project-version/pdf/views/header.jsx]: OIC, so the StatusBanner returns `null` if active - as you were :)

**Comment by joefitter** [comment]
looks fine, one minor stylistic whinge, but other than that g2g

File: pages/project-version/pdf/index.jsx (hunk 1 above)
Comment (wheelsandcogs) [reply pages/project-version/pdf/index.jsx]: Yeah had a bit of a brain fart there, not sure why I used a function.

**Pull request merged**